terraform/modules/route53_balancer/lambda/asg_ip_attach/index.py

```diff
@@ -0,0 +1,30 @@
+import os, boto3, json, re
+
+def handler(event, context):
+    ec2 = boto3.resource('ec2')
+    route53 = boto3.client('route53')
+    message = json.loads(event['Records'][0]['Sns']['Message'])
```

Comment:
https://github.com/hashbang/admin-tools/pull/63/files/0b2fefab2d1f7f15d3cb0c19e742495cd4630186#diff-39ceb2360f3ad2c62e6b1444254fe8d6R114

That is the IAM policy which is only made available to the AWS autoscale group API to report to it.

It is whitelist only, and nothing else is given write access to it.